docs/agent/changelog.md

```diff
@@ -6,6 +6,20 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.15.2 - \[2024-09-09\]
```

Comment:
I _think_ we're going with `3.16.0`, let me see if I can find the thread :waiting

Reply:
https://ngrok.slack.com/archives/C07K6H5Q43C/p1725684469737379?thread_ts=1725670473.245759&cid=C07K6H5Q43C